src/app/components/MarketDataHeader.tsx

```diff
@@ -26,6 +27,10 @@ export default function MarketDataHeader() {
     dispatch(getGlobalData());
   }, [dispatch, code]);
 
+  if (isLoading || data.active_cryptocurrencies === 0) {
+    return <MarketDataHeaderSkeleton />;
+  }
```

Comment:
Let's say isLoading is false, the loading has finished and the final result in API call the active_cryptocurrencies are 0. In this case also will you show the skeleton, I think should we show something like "No results found"?